misc/python/materialize/feature_benchmark/comparator.py

```diff
@@ -45,19 +57,27 @@ def is_regression(self) -> bool:
         return False
 
 
-class RelativeThresholdComparator(Comparator[float]):
-    def ratio(self) -> float:
-        return self._points[0] / self._points[1]
+class RelativeThresholdComparator(Comparator[Optional[float]]):
+    def ratio(self) -> Optional[float]:
+        if self._points[0] is None or self._points[1] is None:
```

Comment:
When do we run into this case? Should this be logged?

Reply:
If no measurement can be taken for either Mz version, it is not possible to calculate the ratio. This happens in case of the counters if the workload has no user-installed workflows. One sice may report "5" messages in the system and the other side will report `None` . I was wondering if I should just report `0` in this case and not have to handle `None`, but then you get a division by zero error further down :-)